examples/power_spectra_example.py

```diff
@@ -0,0 +1,43 @@
+"""
+===============================
+POWER SPECTRUM OF A TIMESERIES
+===============================
+
+An example showing how to estimate power spectrum of a TimeSeries
```

Comment:
I think a `the` should between `estimate` and `power`. Full stop at the end.

Reply:
Done.